bazel/markdown_to_html/test/markdown_to_html.spec.ts

```diff
@@ -1,8 +0,0 @@
-import {markdownToHtml} from '../markdown_to_html';
-
-describe('markdown to html', () => {
-  it('should transform standard markdown to html', () => {
-    const markdown = `**important**`;
-    expect(markdownToHtml(markdown)).toContain('<strong>important</strong>');
```

Comment:
Could we preserve the separation between index.ts as the bazel program that reads files from disk and the library function that acts on a string? I think this way is easier to test and will make it easier to use the marked transforms from other rules/tools.

Reply:
Yes, I can ensure that is done.